task16.py

```diff
@@ -0,0 +1,24 @@
+# # Задача 16: Требуется вычислить, сколько раз встречается некоторое число X в массиве A[1..N]. 
+# # Пользователь в первой строке вводит натуральное число N – количество элементов в массиве. 
+# # В последующих  строках записаны N целых чисел Ai. Последняя строка содержит число X
+
+# # *Пример:*
+
+# # 5
+# #     1 2 3 4 5
+# #     3
+# #     -> 1
+
+import random
+
+list1 = []
+n = int(input("Введите длинну массива: "))
+s = int(input("Введите искомое число: "))
+for i in range(n):
+    list1.append(random.randint(1, 10))
+print(list1)
+count = 0
+for i in range(len(list1)):
+    if list1[i] == s:
```

Comment:
Можно проходиться напрямую по элементам списка. Или, хотя бы, использовать переменную `n`, не считая длину списка.
```suggestion
for i in list1:
    if  i == s:
```